investment-app/TRINV.Api/TRINV.Application/Commands/Authorization/AuthorizeByCodeCommand.cs

```diff
@@ -94,7 +94,7 @@ public async Task<object> Handle(AuthorizeByCodeCommand request, CancellationTok
 
             //ExpiresUtc = DateTimeOffset.UtcNow.AddMinutes(10),
             // The time at which the authentication ticket expires. A 
-            // value set here overrides the ExpireTimeSpan option of 
```

Comment:
Any idea those changes from where come ?

Reply:
![image](https://github.com/MihaelPavlov/Angular_Projects/assets/49004684/ea429940-2d91-4152-92cf-a541aef10820)
I don't know in my files looks diferent I have not done any changes to this file.